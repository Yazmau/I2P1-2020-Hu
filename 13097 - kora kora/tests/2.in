100
8021592 25253381 40255294 70196651 78271341 94976467 95468927 153604749 196550312 198969655 199900234 203494671 220224451 222640995 227716915 227748689 240787610 247785571 301720299 307902362 325478042 326176677 331983982 340646848 350131917 381900326 387677508 413385225 430743374 436070584 455871273 474424834 483500203 484612531 489207317 490810290 496343489 509592757 517296862 535782614 541692929 544791170 547139825 551263872 554426607 563986165 579131453 579934212 585610450 594301301 603422037 615405232 631969322 634294604 639478942 656205055 664621397 667458665 672709700 694767612 697259960 699743209 701634100 701927539 718727624 721229496 721562519 724767103 725361095 738461074 745266504 770764966 771190136 787082868 791328413 793723875 811989556 825616779 829855535 837401295 844093659 860308520 861195007 866242290 868987090 872407545 873923793 874368771 883668311 886538832 904954555 908215172 917911601 919109190 919312873 932375359 957073220 971756883 982522514 999167944
100
2 8
2 85
2 64
2 87
1
2 16
2 98
1
2 72
2 6
1
2 80
1
1
2 6
1
2 69
2 85
2 27
2 3
1
2 85
2 30
1
2 11
1
2 18
2 64
2 39
1
1
2 27
1
2 55
2 67
1
1
2 41
1
1
2 24
2 27
2 53
1
1
2 36
2 22
1
1
2 75
2 76
1
2 30
2 56
2 35
1
2 7
1
2 65
1
2 40
2 51
1
2 52
2 61
2 38
1
1
2 53
2 21
1
2 29
2 50
2 24
2 33
2 8
1
2 45
2 71
1
2 46
1
1
1
2 39
1
2 16
2 47
2 23
1
2 19
2 56
2 50
1
1
1
1
2 31
1
2 22
