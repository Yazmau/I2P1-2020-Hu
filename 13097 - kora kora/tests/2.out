153604749
434493545
175481884
109240474
14234293
30703828
12298169
742003
3364042
185500
772781
441872
94647
4913
55433
13307
3050
1890
2941
986
369
344
197
64
20
11
10
3
1
1
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
